.vscode/extensions.json

```diff
@@ -0,0 +1,364 @@
+/*
+ * Copyright 2022-2023 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+[
+    {
+        "identifier": {
+            "id": "visualstudioexptteam.intellicode-api-usage-examples",
+            "uuid": "9fa2a00e-3bfa-4c2a-abc4-a865bb2b5cf3"
+        },
+        "version": "0.2.7",
+        "location": {
+            "$mid": 1,
+            "path": "/home/ec2-user/.vscode-server/extensions/visualstudioexptteam.intellicode-api-usage-examples-0.2.7",
```

Comment:
I don't think these references will work. Maybe we should leave this file out of version control?

Reply:
File removed and README updated